tests/ttnn/unit_tests/operations/test_moreh_norm.py

```diff
@@ -196,34 +289,103 @@ def test_moreh_norm(input_shape, p, dim_rtol_atol, keepdim, device):
 )
 @pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
 def test_moreh_norm_compute_kernel_options(input_shape, p, dim_rtol_atol, compute_kernel_options, device):
+    """
+    Parametrized test for Tenstorrent's norm operation with various kernel compute options. Compares the output of Tenstorrent's norm
+    with PyTorch's norm for different input shapes, dimensions, norms, and compute kernel configurations.
+
+    Args:
+        input_shape (list of int): Shape of the input tensor.
+        p (float): The order of the norm.
+        dim_rtol_atol (list): List containing the dimension(s), relative tolerance, and absolute tolerance for the comparison.
+        compute_kernel_options: Configuration options for the compute kernel.
+        device: The device to run the computation on.
+
+    Raises:
+        AssertionError: If the computed outputs from Tenstorrent's implementation and PyTorch are not close.
+    """
     torch.manual_seed(2024)
     dim, rtol, atol = dim_rtol_atol
     run_moreh_norm(input_shape, p, dim, rtol, atol, device, compute_kernel_options=compute_kernel_options)
 
 
-@pytest.mark.parametrize("p", [2.0], ids=["p=2.0"])
+@pytest.mark.parametrize("p", [2.0, 2.5, -2.5])
```

Comment:
```suggestion
@pytest.mark.parametrize("p", [2.0])
```
Even if the p-values change, the callback-related logic is unlikely to be affected. There's no need to triple the number of tests.

Reply:
I agree!